### ISSUE TITLE
Invalid function for getter 'WaterArea3D::get_flow_direction' for property: 'flow_direction'.

### ISSUE THREAD

**Issue by sandsalamand** [godot-extended-libraries/hydro#19]
Using Godot 4.1.2, I get this error upon opening the editor and every time I run the game.

```
E 0:00:00:0654   ClassDB::add_property: Invalid function for getter 'WaterArea3D::get_flow_direction' for property: 'flow_direction'.
  <C++ Error>    Condition "mb_get->get_argument_count() != exp_args" is true.
  <C++ Source>   core\object\class_db.cpp:1065 @ ClassDB::add_property()
```

**Comment by jonri** [comment]
I've been out of town but I'll get a chance to take a look later this week for sure.

**Comment by fire** [comment]
I can do some updates for Godot Engine 4.2 compatibility in the next three hours.

**Comment by fire** [comment]
Made a pr https://github.com/godot-extended-libraries/hydro/pull/20

**Comment by jonri** [comment]
Thanks for the PR @fire; @sandsalamand I verified that this is fixed on both 4.1 and 4.2.
